python/README.md

```diff
@@ -21,9 +21,9 @@ I try to make the syntax compatible with `Jinja2`.
 - [ ] template rich printing
 - [x] implement component syntax `{% Component * arg1 arg2 kwarg1=1 kwarg2=2 **kwargs %}`
 - [ ] allow more corner cases for the component syntax
-  - [ ] `{% Componnet arg=" * " %}`
-  - [ ] `{% Componnet arg = " * " %}`
-  - [ ] `{% Componnet arg = await f()`
+  - [x] `{% Component arg=" * " %}`
+  - [x] `{% Component arg = " * " %}`
+  - [ ] `{% Component arg = await f()`
```

Comment:
What does this line `{% Component arg = await f()` mean?

Reply:
`await` is a keyword in programming language like Python and JavaScript.

I think inline expressions inside component tags like

- `{% Component key = not value %}`
- `{% Component number = 1 / 2 * 3 %}`

should work too. Since it is far more complex, which may need parsing an AST, this could be implemented in another PR though. I would merge this if I have time to test this PR tomorrow.

Reply:
If we want to support such grammar, then we are not going to use white space as separator anymore. Maybe we can use comma (,) to separate parameters just like how Python does.

Reply:
See https://github.com/QZGao/core/commit/5569bdd7607aa5a9157ef3c2546cb80a665248cb for "comma-separated arguments" version.